randovania/games/prime1/generator/bootstrap.py

```diff
@@ -71,6 +71,21 @@ def prime1_progressive_damage_reduction(self, db: ResourceDatabase, current_reso
 
         return dr
 
+    def prime1_additive_damage_reduction(self, db: ResourceDatabase, current_resources: ResourceCollection):
```

Comment:
mypy

Reply:
Was this in reference to your second comment because I don't understand what is meant by this otherwise?

Reply:
![image](https://github.com/randovania/randovania/assets/38186597/b888a9b9-0a67-4035-a719-ad4fcb56ec8b)
mypy is a python type checker, and it complains here that this function does not specify a return type.

Reply:
None of the other functions of the same type have a return type, either.

Reply:
we've introduced mypy relatively late, so a lot of early code will still give these errors.
You dont have to change other non-related-to-your-PR functions, but feel free to do it if you want to if they're in the same file.